client/components/animation-slider/index.js

```diff
@@ -0,0 +1,72 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, createRef } from '@wordpress/element';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import './style.scss';
+
+class AnimationSlider extends Component {
+	constructor() {
+		super();
+		this.state = {
+			animate: null,
+		};
+		this.container = createRef();
+		this.onExited = this.onExited.bind( this );
+	}
+
+	onExited() {
+		const { onExited, focusOnChange } = this.props;
+		if ( onExited ) {
+			onExited();
+		}
+		if ( focusOnChange ) {
+			const focusable = this.container.current.querySelector(
+				'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+			);
+			if ( focusable ) {
+				focusable.focus();
+			}
+		}
+	}
+
+	render() {
+		const { children, animationKey, animate } = this.props;
+		const args = { navRight: this.navRight, navLeft: this.navLeft };
```

Comment:
What do navRight/navLeft do here? seems to they both would be null/undefined and i don't see refs to it elsewhere in the code.

Reply:
Oops, thanks. Thats leftover code from the previous attempt